src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -53,5 +61,17 @@
         </iframe>
       </div>
     </div>
+    <div class="col my-3"
+         *ngIf="showMessage">
+      <cd-alert-panel type="info"
+                      class="mb-3"
+                      *ngIf="showMessage"
+                      dismissible="true"
+                      (dismissed)="showMessage = false"
+                      i18n>If the embedded Grafana Dashboard was blocked due to a certificate error, please click <a [href]="grafanaSrc"
```

Comment:
This message is assuming that the user KNOWS it, but what they'll see is that nothing happens (the first time they try this, they don't even know that they should see here a Grafana iframe). So let's adopt the new user standpoint and say something like: "If no embedded Grafana Dashboard appeared below, please follow this link to check if Grafana is reachable and there are no HTTPS certificate issues. You may need to reload this page after accepting any Browser certificate exceptions".